testobserve/tests/src/test/resources/listener_tests/ballerina.conf

```diff
@@ -0,0 +1,3 @@
+[b7a.observability.tracing]
```

Comment:
We don't need this file I think. Shall we remove ?

Reply:
Fixed with https://github.com/ballerina-platform/module-ballerinai-observe/pull/1/commits/7c73daf162634fe0dc248406d04ff6f96d454306